CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Master (Unreleased)
 
+- Update `RSpec/IteratedExpectation` cop documentation with an example of a matcher relying on block arguments. ([@jprince])
```

Comment:
No change log entry is required because the change is a document improvement and have no user impact. Can you remove this?

Reply:
Done